src/sentry/tasks/reports.py

```diff
@@ -705,6 +705,14 @@ def deliver_organization_user_report(timestamp, duration, organization_id, user_
     user = User.objects.get(id=user_id)
 
     if not user_subscribed_to_organization_reports(user, organization):
+        if features.has("organizations:weekly-report-debugging", organization):
+            logger.info(
```

Comment:
If you don't mind doing something hacky I'd recommend: 
```py
log_level = "debug"
if features.has("organizations:weekly-report-debugging", organization):
	log_level = "info"
...
getattr(logger, log_level)(...)
```

Reply:
Eh, I'm going to take it out in a few days anyway